specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-06-01/Accounts.json

```diff
@@ -0,0 +1,1569 @@
+          "readOnly": true
+        }
+      },
+      "type": "object"
+    },
+    "MediaServiceUpdate": {
+      "description": "A Media Services account update.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "properties": {
+          "$ref": "#/definitions/MediaServiceProperties",
+          "description": "The resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "identity": {
+          "$ref": "#/definitions/MediaServiceIdentity",
+          "description": "The Managed Identity for the Media Services account."
+        }
+      },
+      "type": "object"
+    },
+    "ListEdgePoliciesInput": {
+      "properties": {
+        "deviceId": {
+          "type": "string",
+          "description": "Unique identifier of the edge device."
+        }
+      },
+      "type": "object"
+    },
+    "EdgeUsageDataEventHub": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Event Hub where usage will be reported."
+        },
+        "namespace": {
+          "type": "string",
+          "description": "Namespace of the Event Hub where usage will be reported."
+        },
+        "token": {
+          "type": "string",
+          "description": "SAS token needed to interact with Event Hub."
+        }
+      },
+      "type": "object"
+    },
+    "EdgeUsageDataCollectionPolicy": {
+      "properties": {
+        "dataCollectionFrequency": {
+          "type": "string",
+          "description": "Usage data collection frequency in ISO 8601 duration format e.g. PT10M , PT5H."
+        },
+        "dataReportingFrequency": {
+          "type": "string",
+          "description": "Usage data reporting frequency in ISO 8601 duration format e.g. PT10M , PT5H."
+        },
+        "maxAllowedUnreportedUsageDuration": {
+          "type": "string",
+          "description": "Maximum time for which the functionality of the device will not be hampered for not reporting the usage data."
+        },
+        "eventHubDetails": {
+          "$ref": "#/definitions/EdgeUsageDataEventHub",
+          "description": "Details of Event Hub where the usage will be reported."
+        }
+      },
+      "type": "object"
+    },
+    "EdgePolicies": {
+      "properties": {
+        "usageDataCollectionPolicy": {
+          "$ref": "#/definitions/EdgeUsageDataCollectionPolicy"
+        }
+      },
+      "type": "object"
+    },
+    "MediaServiceIdentity": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The identity type."
+        },
+        "principalId": {
+          "type": "string",
+          "format": "uuid",
```

Comment:
@pilor  Can you give us more info on this uuid usage issue.
https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#R3017
According to this error, it says ARM discourages usage of GUID. but doesn't give much more info on it.

Reply:
Guid input is discouraged since it can be hard for users to generate a proper GUID.  This case is fine since its readonly.  However, if you look at other implementations of this contract its not generally defined as an explicit uuid format.  Up to you whether you keep it defined tightly as a uuid or not

Reply:
Thank you.